packages/manager/modules/telecom-spare/src/modems/order/modems-order.html

```diff
@@ -0,0 +1,236 @@
+            <div class="col">
+                <!-- ORDER -->
+                <section class="widget-presentation"
+                         data-ng-if="$ctrl.order.orderURL">
+                    <h2 class="widget-presentation-title" data-translate="modems_order_shipping_title"></h2>
+                    <div class="alert alert-success" role="alert" data-translate="modem_order_order_success"></div>
+                    <p>
+                        <a target="_blank" class="oui-button oui-button_primary" rel="noopener"
+                            data-ng-href="{{:: $ctrl.order.orderURL }}">
+                            <span data-translate="modem_order_order_url"></span>
+                            <i class="fa fa-external-link" aria-hidden="true"></i>
+                        </a>
+                    </p>
+                </section>
+
+                <!-- SHIPPING CONTACT AND MODEM SELECTION -->
+                <section class="widget-presentation"
+                        data-ng-if="$ctrl.orderStep === 'modem_and_shipping' && !$ctrl.order.success">
+                    <div class="row">
+                        <div class="col-sm-6">
+                            <h2 class="widget-presentation-title" data-translate="modems_order_shipping_title"></h2>
+                            <div>
+                                <h3 data-translate="modems_order_contact_shipping_title"></h3>
+                                <ovh-contact data-ng-model="$ctrl.order.contact"
+                                                data-ovh-contact-choice-options="$ctrl.contactChoiceOptions"
+                                                data-ovh-contact-init-deferred="$ctrl.contactDeferred">
+                                    <oui-spinner></oui-spinner>
+                                </ovh-contact>
+                            </div>
+                            <div>
+                                <h3 data-translate="modems_order_shipping_mode_title"></h3>
+                                <tuc-shipping-mode-selection data-ng-model="$ctrl.order.shipping.mode"
+                                                            data-tuc-shipping-mode-selection-relay="$ctrl.order.shipping.relay"
+                                                            data-tuc-shipping-mode-selection-options="$ctrl.order.shipping.options">
+                                </tuc-shipping-mode-selection>
+                            </div>
+                        </div>
+                        <div class="col-sm-6">
+                            <h2 class="widget-presentation-title" data-translate="modems_order_modem_title"></h2>
+                            <div>
+                                <h3 data-translate="modems_order_select_title"></h3>
+                                <select class="form-control" data-ng-model="$ctrl.order.brand">
+                                    <option data-ng-repeat="brand in $ctrl.brands" value="{{brand.id}}">{{brand.name}}</option>
```

Comment:
```suggestion
                                    <option data-ng-repeat="brand in $ctrl.brands track by band" value="{{brand.id}}">{{brand.name}}</option>
```

Also can't a `oui-select` be used ?

Reply:
is it possible with an object which on you display a value and you get the key ?

Reply:
https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/select